Examples/src/main/java/com/mapbox/vision/examples/ExternalVideoSourceActivity.java

```diff
@@ -195,6 +201,15 @@ private void startFileVideoSource() {
                         ImageFormat.RGBA,
                         imageSize
                 );
+                // set CameraParameters them each time they change
```

Comment:
I'd add the comment exactly here ("...that this features still won't work for this particular example...")

Reply:
Well, ok